gradlew

```diff
@@ -1,164 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Why are we removing gradlew and adding the bat version? I often develop on os x and linux (and cygwin on windows) - this needs to be restored.  We should have both.